src/cloudkit.cs

```diff
@@ -1696,12 +1623,6 @@ interface CKSubscription : NSSecureCoding, NSCopying {
 		[Export ("predicate", ArgumentSemantic.Copy)]
 		NSPredicate Predicate { get; }
 
-		[NoWatch]
-		[Deprecated (PlatformName.iOS, 10,0, message: "Use 'CKQuerySubscriptionOptions'.")]
-		[Deprecated (PlatformName.MacOSX, 10,12, message: "Use 'CKQuerySubscriptionOptions'.")]
-		[Export ("subscriptionOptions", ArgumentSemantic.UnsafeUnretained)]
```

Comment:
This still seems valid if you're targeting iOS 8 - 9.

Reply:
The submission was complaining about this. Was not deleted due to the deprecation warning.